src/index.html

```diff
@@ -8,10 +8,172 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+      class="main__form"
+    >
+
+      <fieldset class="fieldset">
+        <legend> Personal information </legend>
+
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            class="enter"
+            name="surname"
+            autocomplete="off"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+           type="text"
+           class="enter"
+           name="name"
+           autocomplete="off"
+           required>
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            class="enter"
+            min="1"
+            max="100"
+            value="12"
+            name="age">
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            class="enter"
+            name="birthDate">
+        </label>
+
+        <label class="form-field lastInput">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            class="enter"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend> Registration </legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            class="enter"
+            name="mail"
+            placeholder="email@example.com"
+            required>
+        </label>
+
+        <label class="form-field lastInput">
+          Password:
+          <input
+            type="password"
+            class="enter"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            required>
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend> An interesting fact about you! </legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <input
+            type="radio"
+            class="enter"
+            name="cat"
+            value="y"> Yes
```

Comment:
Let's put text on a newline to improve code readability, do it for the next radio too
```suggestion
            value="y"> 
            Yes
```